src/pages/AboutPage.tsx

```diff
@@ -1,8 +1,59 @@
-import React from "react";
+import React, { useState } from "react";
+import Button from "@material-ui/core/Button";
 import { Header } from "components/Header";
 
 export const AboutPage: React.FC = () => {
+  const [aboutData, setAboutData] = useState({
+    server: "openrainbow.com",
+    telephonyVersion: "unknown",
+    systemIdentifier: "PBX7467-...",
+    CopyrightYear: "2021",
+  });
   return (
-    <Header title="Rainbow Click2Cal" subtitle="1.7.20" toPage="settings" />
+    <div>
+      <Header
+        title="Rainbow Click2Cal"
+        subtitle="Version: 1.7.20"
+        toPage="settings"
+      />
+      <div style={{ padding: "15px" }}>
+        <p>Server: {aboutData.server}</p>
+        <p>Telephony agent version: {aboutData.telephonyVersion}</p>
+        <p>System identifier: {aboutData.systemIdentifier}</p>
+
+        <p style={{ padding: "15px 0" }}>
+          © Copyright Alcatel-Lucent Enterprise, {aboutData.CopyrightYear}
+        </p>
+
+        <div style={{ marginBottom: "15px" }}>
+          For more information about Rainbow, visit
+          <a
```

Comment:
I'm getting a warning in the compiler `Using target="_blank" without rel="noreferrer" is a security risk: see https://html.spec.whatwg.org/multipage/links.html#link-type-noopener`

You should always handle warnings

Also, always remember to follow the DRY principle (Don't repeat yourself)

All these anchor elements have the same `style` and `target` properties repeated multiple times.

It makes sense to extract this element out and turn it into a subcomponent like this

```tsx
interface AboutLinkProps {
  url: string;
  label: string;
}

const AboutLink: React.FC<AboutLinkProps> = ({ url, label }) => {
  return (
    <div>
      <a target="_blank" rel="noopener noreferrer" href={url}>
        {label}
      </a>
    </div>
  );
};
```

and then to use it like this

```tsx
<AboutLink
  url="https://openrainbow.com/legals/terms-of-service"
  label="Terms of Use"
/>
```